warehouse/cli/search.py

```diff
@@ -10,11 +10,24 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import click
+
 from warehouse.cli import warehouse
+from warehouse.search.tasks import reindex as _reindex
 
 
 @warehouse.group()  # pragma: no branch
 def search():
     """
     Manage the Warehouse Search.
     """
+
+
+@search.command()
+@click.pass_obj
+def reindex(config):
+    """
+    Recreate the Search Index.
+    """
+
+    config.task(_reindex).apply_async()
```

Comment:
@dstufft I couldn't figure out how to get this to run "in the foreground".

I tried wrapping `apply()` in a way similar to `warehouse.tasks:WarehouseTask.apply_async()` but kept getting `transaction.interfaces.NoTransaction`

Reply:
runs great on the worker though!

Reply:
You probably have to do some setup to get the request in a reasonable state.